lib/ae_bank_days/bank_day_helper.rb

```diff
@@ -11,7 +11,7 @@ def bank_day?(day)
       end
 
       def next_banking_day(date, number_of_days: 0)
-        banking_day = date
+        banking_day = date.to_date
```

Comment:
This method will need to be updated to fix the bug where it can return the current day if `number_of_days` is `0`